Disables holiday stuff, to tempfix a shitty PR

Altoids1 opened the pull request:
### github.com/tgstation/tgstation/pull/31877

Apparently this let people be spooky-ass fuckin races (including _shadowling_) on roundstart for every day of October, which, eh, naawww, fam. **This is a temporary fix** while Highway or whoever figures out how to fix this.

AdamElTablawy commented:
Make your changelogs actually describe what's happening so people looking at it can tell what it means.

Super3222 commented:
Could you make the changelog less memey.

Altoids1 committed:
Disables holiday stuff, to counter a shitty PR

github.com/tgstation/tgstation/pull/31877

Apparently this let people be spooky-ass fuckin races on roundstart for every day of October, which, eh, naawww, fam.

Commit:
YOGS

YOGS
